### ISSUE TITLE
Implementing an Uberischt widget for displaying inline docstrings in skhdrc files

### ISSUE THREAD

**Issue by mstr3336** [koekeishiya/skhd#110]
Hi, I'm planning on writing a simple widget that parses inline docstrings (Probably regular comments with a leading character to indicate its a docstring, like `#'` or something). 

I was wondering if, once done, you'd be open to linking to it from the readme? 
Also, I was wondering if you had suggestions for the syntax, I was thinking something like:

```
#' ${cmd} + alt - 1 : Go to workspace 1
#' ${cmd} + alt - n : Go to workspace n
```
https://github.com/koekeishiya/skhd/blob/01722fcf49a5826392010504162bc63e2b01c423/examples/skhdrc#L170-L179

Or something, where cmd is escaped one way or another and automatically parsed to &#8984;

& perhaps following markdown syntax in all other regards.

**Comment by koekeishiya** [comment]
Is the intention for this to be some kind of "cheat-sheet" to quickly look up configured binds and what they do?

**Comment by mstr3336** [comment]
> Is the intention for this to be some kind of "cheat-sheet" to quickly look up configured binds and what they do?

More or less

**Comment by deekue** [comment]
looking to do the same thing.  

I borrowed the format from [Regolith's Remontoire](https://github.com/regolith-linux/remontoire) and built an initial [skhdrc for Yabai](https://github.com/deekue/dotfiles/blob/master/config/skhd/yabai_skhdrc)(still a WIP).

Haven't looked at writing the Uberischt widget yet.

**Comment by mstr3336** [comment]
> looking to do the same thing.
> 
> I borrowed the format from [Regolith's Remontoire](https://github.com/regolith-linux/remontoire) and built an initial [skhdrc for Yabai](https://github.com/deekue/dotfiles/blob/master/config/skhd/yabai_skhdrc)(still a WIP).
> 
> Haven't looked at writing the Uberischt widget yet.

Yeah, regoliths implementation is great- looking forward to seeing what you do!

**Comment by deekue** [comment]
[first pass](https://github.com/deekue/uebersicht-remontoire.widget) is clunky but it works.

it'd be better to get the current config from skhd itself (eg. Remontoire makes an IPC to i3).  It would also cover the case of any `.load` statements  

something like  
`skhd --dump_config`

**Comment by deekue** [comment]
I changed tack and refactored [skhd_parse.py](https://github.com/deekue/dotfiles/blob/macos_tiling_wm/bin/skhd_parse.py) to output HTML directly or a base64 encoded data: URI, which can be passed to Google Chrome.  

```
# ## Launch // This Dialog // <fn><shift> ? ##
fn + shift - 0x2C : open -na "Google Chrome" --args --new-window --app="$(skhd_parse.py -f data)"
```
